packages/editor/src/hooks/template-part-edit.js

```diff
@@ -84,6 +73,6 @@ export const withEditBlockControls = createHigherOrderComponent(
 
 addFilter(
 	'editor.BlockEdit',
-	'core/edit-site/template-part-edit-button',
+	'core/editor/template-part-edit-button',
 	withEditBlockControls
 );
```

Comment:
What's (which dependency) preventing us from bundling this code within the template part block code directly?

Reply:
Nothing stands out to me. I'll try it.

Reply:
👍 27962d05c5